civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -76,6 +76,17 @@
 * TEMP fixes till ui-select styling is fixed
 * Dropdown ul opacity was set to 0
 */
-.chr_leave-request-modal__form .ui-select-container ul {
-  opacity: 1 !important;
+.chr_leave-request-modal__form {
+ .ui-select-container ul {
+   opacity: 1 !important;
+  }
+
+  .dropdown-menu {
+    border: 1px solid #ccc;
+  }
+
+  .ui-select-container:not(.ui-select-multiple) input {
+    /*the width was getting set to 503px in element itself thus causing the input box to reduce*/
+    width: 100% !important;
```

Comment:
In manager leave page's Filter section and in calendar, UI Select is used. They are not broken by this change right?